lib/ipv6.js

```diff
@@ -543,7 +539,7 @@ Address6.prototype.correctForm = function () {
   });
 
   if (zeroes.length > 0) {
-    var index = zeroLengths.indexOf(max(zeroLengths));
+    var index = zeroLengths.indexOf(Math.max.apply(Math, zeroLengths));
```

Comment:
spread in function calls has the same availability as all of the other features you're using in this PR and `Math.max(...zeroLengths)` is much more readable so let's use that :)

Reply:
Fine by me but it’s a relatively new feature and I wasn’t sure if it would have an adverse effect on runtime support (I didn’t check).

Reply:
Ok, it looks pretty well supported https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Browser_compatibility